email_campaigns/email_to_mentors.txt

```diff
@@ -0,0 +1,34 @@
+Subject: You’re mentoring for the mentored sprint at PyCon US!
+
+{Full name},
+
+Thank you for signing up to mentor for the {project} at the mentored sprint for diverse beginners at PyCon US.
+We’re pleased to welcome you to the sprint.
+
+The mentored sprint will take place on **Saturday, May 4th 2:30 PM - 6:30 PM. Room 25C**
+
+Some useful information:
+
+- The mentored sprint is organized as part of PyCon US. You will need to be registered
+  for PyCon in order to attend the mentored sprint. If you have not registered, please do
+  so as soon as possible, because PyCon is a sell out year after year.
+ 
+- We're excited that numerous projects and mentors have signed up for the sprint! Each
+  project will have the opportunity to pitch for their own project to the group. Please
+  be prepared with a **3 minute pitch**.
+
+- We understand that setting up local environment can be time consuming, and the steps
+  can vary from one project to another. To help the event more successful, we recommend
+  that you include detailed setup instruction in your project's contributing guideline well
+  in advance.
+
+- We also recommend that you prepare a set of beginner friendly issues to be worked on
+  during the sprint.
+
+By attending the mentored sprint at PyCon, you are also accepting to uphold PyCon and
+The PSF’s code of conduct. Please take a moment to read it through at https://us.pycon.org/2019/about/code-of-conduct/
+
+Thank you again for your time to mentor! We are grateful for your work in open source and Python.
+See you soon in Cleveland! Please do not hesitate to reach us if you have additional questions.
```

Comment:
Please do not hesitate to reach us if you have additional questions or would like an extra bit of help preparing your project for the sprint.